primitives/state-machine/src/overlayed_changes.rs

```diff
@@ -498,6 +510,34 @@ impl OverlayedChanges {
 		)
 	}
 
+	/// Get an iterator over all child tries that are stored in the overlay.
```

Comment:
```suggestion
	/// Get an iterator over all pending and committed child tries in the overlay.
	///
	/// NOTE: The returned iterator can return the same child trie multiple times.
```

Reply:
Shouldn't we rather just de duplicate the child infos here in this function?

Reply:
I opted for de de duplicating them in the `fn child_infos()` as the duplicates would go away with the storage transaction changes anyways.